src/benchmark/file_io_write_micro_benchmark.cpp

```diff
@@ -14,19 +14,48 @@ const int32_t MB = 1000000;
 
 class FileIOWriteMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
  public:
+  uint64_t control_sum = uint64_t{0};
+
   void SetUp(::benchmark::State& state) override {
     // TODO(phoeinx): Make setup/teardown global per file size to improve benchmark speed
     ssize_t BUFFER_SIZE_MB = state.range(0);
     // each int32_t contains four bytes
     int32_t vector_element_count = (BUFFER_SIZE_MB * MB) / 4;
     data_to_write = std::vector<int32_t>(vector_element_count, 42);
+    control_sum = vector_element_count * uint64_t{42};
 
     if (creat("file.txt", O_WRONLY) < 1) {
       std::cout << "create error" << std::endl;
     }
     chmod("file.txt", S_IRWXU);  // enables owner to rwx file
   }
 
+  void SanityCheck(int32_t NUMBER_OF_BYTES){
```

Comment:
Naming Scheme for functions is lower case with words separated by an underscore.
(I think we have to conform to Googles naming for functions for google benchmark provided functions like `TearDown()` and `SetUp()`, but everything else should follow hyrise naming conventions.